cpp-terminal/key.cpp

```diff
@@ -150,3 +201,89 @@ char Term::Key::toupper()
   else
     return static_cast<char>(m_value);
 }
+
+bool Term::Key::isunicode() const  //FIXME not check for bad unicode
```

Comment:
i was wondering if my utf8::Char that I use in the Lime editor could help here.

Reply:
Would have a try maybe we can include in other PR. This is mainly just remove out the spciale Key F1 F2 etc..